src/assets/styles/multiselect.css

```diff
@@ -0,0 +1,33 @@
+@import '../../../node_modules/vue-multiselect/dist/vue-multiselect.css';
```

Comment:
Would be better to change this to `@import 'vue-multiselect/dist/vue-multiselect.css';`

As preparation for the monorepo, nothing should point directly to the `node_modules` directory and instead be resolved via the module resolver of the build system. In this case it's `vite` which automatically resolves this. However, it is sometimes necessary (depending on where main points in the package.json) to adjust the directory like this: `@import 'vue-multiselect/vue-multiselect.css';`

Reply:
Ah thanks, have fixed